uint16 msecs_to
---
uint8 confirm